source/_docs/free-https.md

```diff
@@ -0,0 +1,205 @@
+    <pre><code>terminus remote:wp &lt;site&gt;.&lt;env&gt; -- plugin install lh-hsts --activate</code></pre>
+    <p>Once enabled, the following header will be sent in responses:</p>
+    <pre><code>Strict-Transport-Security: max-age=15984000; includeSubDomains; preload</code></pre>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="drops7">
+    <ol>
+      <li>Install the <a href="https://drupal.org/project/hsts">HTTP Strict Transport Security</a> module using the <a href="https://www.drupal.org/docs/7/extending-drupal/installing-modules">Drupal interface</a> or with <a href="/docs/terminus">Terminus</a>:
+      <pre><code>terminus remote:drush &lt;site&gt;.&lt;env&gt; -- pm-enable hsts --yes</code></pre>
+      </li>
+      <li>Visit the module configuration page (<code>/admin/config/security/hsts</code>).</li>
+      <li>Check the <strong>Enable HTTP Strict Transport Security</strong> checkbox, set <strong>Max Age</strong> to <code>15552000</code> and click <strong>Save Configuration</strong>.</li>
+    </ol>
+    <p>Once installed and configured, the following header will be sent in responses:</p>
+    <pre><code>strict-transport-security: max-age=15552000</code></pre>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="drops8">
+    <ol>
+      <li>Install the <a href="https://drupal.org/project/hsts">HTTP Strict Transport Security</a> module using the <a href="https://www.drupal.org/docs/8/extending-drupal-8/installing-modules">Drupal interface</a> or with <a href="/docs/terminus">Terminus</a>:
+      <pre><code>terminus remote:drush &lt;site&gt;.&lt;env&gt; -- pm-enable hsts --yes</code></pre>
+      </li>
+      <li>Visit the module configuration page (<code>/admin/config/system/hsts</code>).</li>
+      <li>Check the <strong>Enable HTTP Strict Transport Security</strong> checkbox, set <strong>Max Age</strong> to at least <strong>1 year</strong> and click <strong>Save Configuration</strong>.</li>
+    </ol>
+    <p>Once installed and configured, the following header will be sent in responses:</p>
+    <pre><code>strict-transport-security: max-age=31536000</code></pre>
+  </div>
+</div>
+
+## Compare to Legacy HTTPS
+<table class="table  table-bordered table-responsive">
+  <thead>
+    <tr>
+      <th></th>
+      <th>Legacy</th>
+      <th>Free & Automated</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Price</th>
+      <td>$30/mo per environment</td>
+      <td>Free for all environments</td>
+    </tr>
+    <tr>
+      <th>Certificate type</th>
+      <td>Bring your own</td>
+      <td>Shared, issued by Let's Encrypt</td>
+    </tr>
+    <tr>
+      <th>Renewal</th>
+      <td>Manual</td>
+      <td>Automatic</td>
+    </tr>
+    <tr>
+      <th>Unique inbound IP Address</th>
```

Comment:
They will still have an A record to point to on "Free & Automated"
Maybe something like "Inbound IP: Unique   |  Shared"